float64 width # gripper
int32 mode # mode: 0 -grasp, 1-release

